google/ggrc-core
# GGRC-6032/GGRC-6111/GGRC-6274/GGRC-6288 Update objects related mappings config

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #8508

# Issue description

Allow to search objects by mapping to a person.
Allow to map directives against each other.
Exclude Assessment Template from mapping filter for all objects except Audit (AT can be mapped on Audit only)
Exclude Document from mapping filter for Audit and Assessment
Exclude Program and Assessment from mapping filter for Program (mapping is prevented)
Exclude Access Group from mapping filter for Access Group (mapping is prevented)
Exclude Audit from mapping filter for Audit (mapping is prevented)
Exclude Assessment from mapping filter for Assessment
Exclude Assessment, Audit, Document from mapping filter for Document.

# Steps to test the changes

1. Log in GGRC and create a standard
2. Click on Add tab and look at the dropdown list: Regulations, Policies, Contracts are missed

_Actual Result:_ Cannot map directives (Regulations, Policies, Contracts to Standard) to Standard, except Requirements.
_Expected Result:_ User should be able to map Regulations, Policies, Contracts to Standard

# Solution description

Update mappings config.

# Sanity checklist




# PR Review checklist
